src/platform/testing/e2e/cypress/support/commands/expandAccordions.js

```diff
@@ -1,13 +1,53 @@
 /**
- * Expands all accordions and AdditionalInfo components.
+ * Expands all Accordions and AdditionalInfo components.
+ * Web Components that require Shadow DOM broken out from React Components
  */
 Cypress.Commands.add('expandAccordions', () => {
   Cypress.log();
 
   cy.get('main').then($main => {
-    const accordions = $main.find('button[aria-expanded=false]');
-    if (accordions.length) {
-      cy.wrap(accordions).each(button => {
+    // Check if va-accordion-item Web Component exists
+    if ($main.find('va-accordion-item').length > 0) {
+      cy.get('va-accordion-item')
+        .shadow()
+        .then(accordion => {
+          // If it exists and Accordions are not already expanded
+          if (accordion.find('button[aria-expanded=false]').length > 0) {
+            cy.get('va-accordion-item')
+              .shadow()
+              .find('button[aria-expanded=false]')
+              .each(button => {
+                // Click to open Accordions
+                cy.wrap(button).click({ force: true });
+              });
```

Comment:
I'm still not sure why we are doing this same thing three times. Can we run the same iterator on a combined array of all the nodes we want to expand?